### ISSUE TITLE
Limitation of <sdshare:resource> to exactly one per fragment - Why?

### ISSUE THREAD

**Issue by heuer** [SDShare/Specification#46]
5.1.5. says:

> This element [sdshare:resource] MUST occur exactly once as a child element of each entry. 

I wonder why this limitation was introduced. In an older spec it was allowed to mention 1..n subjects

If the subject A and B are modified within a transaction the server has to 
a) produce two fragment entries which point to the same fragment with different resource elements
b) divide the transaction into two parts and produce a fragment which modifies subject A and one fragment which modifies B

I wonder if (b) is realisable and if (a) desirable since the fragment feed becomes longer and the client would perform the deletion process of section 5.2.3. for the same fragment for subject A and B unless it is smart enough to recognize that the Atom entry for A and B point to the same fragment (a client could recognize it if the Atom entry id is the same or if the link is the same).

Basically my question is: What are the reasons for the "exactly once" limitation? :)
